From c99c22b099bff0126af62615ad1901eb21bfd50a Mon Sep 17 00:00:00 2001
From: Konrad Leszczynski <konrad.leszczynski@intel.com>
Date: Tue, 8 Dec 2015 14:51:09 +0000
Subject: [PATCH 27/28] usb: ssic: change config file for the SSIC sideband
 wakeup solution

This change in the x86_64_defconfig file is necessary for the SSIC
sideband wakeup solution (Intel SSIC GPIO Driver) to work. The driver
is enabled by default.

Change-Id: I343636907fe72235bc01b454bc5dd95a745172cf
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-9290
Signed-off-by: Konrad Leszczynski <konrad.leszczynski@intel.com>
Reviewed-on: https://android.intel.com:443/445729
---
 arch/x86/configs/x86_64_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/x86_64_defconfig b/arch/x86/configs/x86_64_defconfig
index 0875fd0..a554cc2 100644
--- a/arch/x86/configs/x86_64_defconfig
+++ b/arch/x86/configs/x86_64_defconfig
@@ -3575,6 +3575,7 @@ CONFIG_INTEL_RST=y
 # CONFIG_PVPANIC is not set
 CONFIG_INTEL_WARN_INT=y
 # CONFIG_INTEL_GPIO_HSIC is not set
+CONFIG_INTEL_GPIO_SSIC=y
 CONFIG_INTEL_PSTORE_PRAM=y
 # CONFIG_INTEL_TASKS_DUMP is not set
 # CONFIG_CHROME_PLATFORMS is not set
-- 
1.9.1

